src/components/CollectionsAndPalettes/Collections/Collection/Collection.tsx

```diff
@@ -279,15 +320,12 @@ class Collection extends React.Component<Props, State> {
         </div>
 
         {/*The commonArea element is added here in order to keep the AddToCart Button inside the Collection Cards container, also decide its position*/}
-        {materials.length > 0 && (
+        {this.collectionMaterials.length > 0 && (
           <div className={"commonArea"}>
-            <MoreIdeas
-              collectionMaterials={finalItems.filter(
-                (item) => item.objectType === "material"
-              )}
-            />
+            <MoreIdeas collectionMaterials={this.collectionMaterials} />
           </div>
         )}
+        {this.hpTags.length > 0 && <ExploreTags buttons={this.hpTags} />}
```

Comment:
Please add a handler to the onButtonSelected event inside this component so we don't get an error when we click the tags.